### PR TITLE
Set the FQDN of the host

### PR DESCRIPTION
**Pull request by dasJ**

### CODE REVIEW

File: tasks/fqdn.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+- name: Write the system's host file
+  template:
+    src: hosts
```
Comment (SlothOfAnarchy) [inline tasks/fqdn.yml]: rename to .j2

File: templates/hosts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+# {{ ansible_managed }}
+
+# localhost
+127.0.0.1 localhost localhost.{{ domain }}
+::1 localhost localhost.{{ domain }}
+
+# IPv4
+{% for addr in ansible_all_ipv4_addresses %}
+{{ addr }} {{ inventory_hostname_short }}.{{ domain }} {{ inventory_hostname_short }}
```
Comment (SlothOfAnarchy) [inline templates/hosts]: indent

File: templates/hosts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+{% endfor %}
+
+# IPv6
+{% for addr in ansible_all_ipv6_addresses %}
+{{ addr }} {{ inventory_hostname_short }}.{{ domain }} {{ inventory_hostname_short }}
```
Comment (SlothOfAnarchy) [inline templates/hosts]: indent

**Comment by dasJ** [comment]
@SlothOfAnarchy Pls rereview

**Pull request merged**